### PR TITLE
Materialization

### PR DESCRIPTION
**Pull request by RussTorres**
Materialize sections using betterboxclient.

### CODE REVIEW

**Comment by RussTorres** [comment]
the materialize schemas might overlap with a file created in #68 ....

File: integration_tests/test_data.py [hunk 1]
```diff
@@ -1,163 +1,173 @@
-import os
```
Comment (fcollman) [inline integration_tests/test_data.py]: I don't understand what happened here... the diff is nuts, I can't see what is new and what isn't..

File: rendermodules/materialize/schemas.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+class MaterializedBoxParameters(argschema.schemas.DefaultSchema):
+    stack = Str(required=True, description=(""))
+    rootDirectory = OutputDir(required=True, description=(""))
+    width = Int(required=True, description="")
+    height = Int(required=True, description="")
+    maxLevel = Int(required=False, default=0, description="")
+    fmt = Str(required=False)  # TODO validate to allow PNG, TIF(F?), JPG
+    maxOverviewWidthAndHeight = Int(required=False, description=(""))
+    skipInterpolation = Boolean(required=False, description=(""))
+    binaryMask = Boolean(required=False, description=(""))
```
Comment (fcollman) [inline rendermodules/materialize/schemas.py]: I think this description as a tuple will break the documentation

File: integration_tests/test_data.py (hunk 1 above)
Comment (RussTorres) [reply integration_tests/test_data.py]: ugh, yes.  I had to reinstall git and might have introduced a line ending mismatch or something.  I can close and roll back commits to try and triage.

File: rendermodules/materialize/schemas.py (hunk 2 above)
Comment (RussTorres) [reply rendermodules/materialize/schemas.py]: not tuples, just empty strings.  Either way, I will need to commit to add documentation.

**Comment by RussTorres** [comment]
Misfire.  Closing until the commit log and documentation is presentable.

**Pull request closed without merging**